textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/dictionarymatcher/DictionaryMatcherTest.java

```diff
@@ -103,6 +126,42 @@ public void testSingleWordQueryInStringFieldUsingScan() throws Exception {
     }
 
     /**
+     * Scenario S-2C:verifies GetNextTuple of DictionaryMatcher and single word
+     * queries in String Field using SCANOPERATOR
+     * Test in Chinese.
+     */
+
+    @Test
+    public void testSingleWordQueryInStringFieldUsingScanChinese() throws Exception {
+
+        ArrayList<String> names = new ArrayList<String>(Arrays.asList("孔明"));
+        IDictionary dictionary = new Dictionary(names);
+
+        // create data tuple first
+        List<Span> list = new ArrayList<Span>();
+        Span span = new Span("firstName", 0, 2, "孔明", "孔明");
+        list.add(span);
+        Attribute[] schemaAttributes = new Attribute[TestConstantsChinese.ATTRIBUTES_PEOPLE.length + 1];
```

Comment:
line 144-148:
There is a new helper function that can do the same thing:
`        Schema resultSchema = Utils.addAttributeToSchema(TestConstantsChinese.SCHEMA_PEOPLE, SchemaConstants.SPAN_LIST_ATTRIBUTE);
`